### DISCUSSION TITLE
【优化自定义】新增一些关键节点的id或者class，方便外部引入者的开发

### DISCUSSION THREAD

**Discussion by Vixcity** [tangly1024/NotionNext#2031] [Ideas]
**为什么提出这个新的特性改动**
『新增一些关键节点的id或者class，方便外部引入者的开发』

**描述一下你推荐的解决方案**
增加 id 或者 class，个人更推荐 class

**Comment by tangly1024** [comment]
很有必要的功能：
- 将在 [feat_div_id ](https://github.com/tangly1024/NotionNext/tree/feat_div_id)分支上进行这次改动；
- 将参考之前您提交的 https://github.com/tangly1024/NotionNext/pull/726 进行改动。
